### DISCUSSION TITLE
Data Layer

### DISCUSSION THREAD

**Discussion by kawpii** [withastro/roadmap#296] [Proposal]
I think since Astro focused on content based websites, it is good to have data layer like Gatsby.

Benefits of data layer is pointed out here : https://www.gatsbyjs.com/blog/scaling-rsg-builds-with-gatsbys-data-layer

While there may be no built in GraphQL support in Astro, there can be otherways to add a data layer. Or support data layer as plugin using GraphQL.

**Comment by delucis** [comment]
Astro’s content layer shipped as stable in v5! https://astro.build/blog/content-layer-deep-dive/
